server/controllers/Followers.js

```diff
@@ -161,6 +176,25 @@ class Followers {
     }
     return false;
   }
+
+  /**
+   * @name UpdateUserFollowersAndFollowingsCount
+   * @async
+   * @static
+   * @memberof Followers
+   * @param {Object} source the source user object
+   * @param {Object} target the target user object
+   * @returns {Null} Null object
+   */
+  static async UpdateUserFollowersAndFollowingsCount(source, target) {
+    const followingsCount = await source.countAllFollowings();
+    const followersCount = await target.countAllFollowers();
+
+    await User.update({ followingsCount }, { where: { id: source.id } });
```

Comment:
Why do we have this attribute on a user?

Reply:
I didn't create a profile table, and I informed you about this when I wanted to start the task that I was only going to extend the existing user table.

As for storing the counts of the followers and followings, there is no provision for followers Count table, and even if there is, the only way I could get it is still by watching for when a user follows someone or is followed by someone, update the counts, and store it in a table.

@dinobi